codes/models/SRRaGAN_model.py

```diff
@@ -46,6 +47,12 @@ def __init__(self, opt):
         if self.is_train:
             # G pixel loss
             #"""
+            # ESRGAN-FS Changes
+            if train_opt['use_frequency_separation']:
```

Comment:
Change it to something like this
```self.use_frequency_separation = train_opt['use_frequency_separation']
            if self.use_frequency_separation:
                self.filter_low = filters.FilterLow().to(self.device)
                self.filter_high = filters.FilterHigh().to(self.device)```

Reply:
Oh, I missed this, will work on it.

Reply:
Wait, I fixed it in a way that conforms more with the rest of the code...
What do you think? Commit below

Reply:
Looks good